MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include pyproject.toml
 recursive-include dimod/include *h
 recursive-include dimod *.pyx *.pxd *.pxi *.pyx.src
+include dimod/py.typed
```

Comment:
Is `py.typed` generated during build, or it should be committed?

Reply:
Needs to be committed apparently. Supposedly we shouldn't need to do this at all post Python 3.4, but testing shows it needs to be there :shrug: